### DISCUSSION TITLE
customize each end point URI

### DISCUSSION THREAD

**Discussion by hyh1618** [thomaxxl/safrs#131] [General]
can you please give an example of change individual service end point's URL?

**Comment by thomaxxl** [comment]
This depends what part of the url you want to change.
If you want to change the api prefix, you have to use the `SAFRSAPI` `prefix` argument.
Example: https://gist.github.com/thomaxxl/dc3890217eea074e561d02988c776eb4#file-gistfile1-txt-L30

If you want to change the collection endpoint name, you have to override `SAFRSBase._s_collection_name`
Example: https://gist.github.com/thomaxxl/dc3890217eea074e561d02988c776eb4#file-gistfile1-txt-L23
